mlir/utils/jenkins/Jenkinsfile

```
@@ -201,11 +201,15 @@ void preMergeCheck(String codepath) {
           ln -s build/compile_commands.json compile_commands.json
         fi
         '''
+        def targetBranch = ${env.CHANGE_TARGET}
```

not sure if this will work since we use env.CHANGE_TARGET outside of sh script here
if it doesn't work try:

```suggestion
        def targetBranch = env.CHANGE_TARGET
```
if it works as it is, great

---

i see it's fixed now with quotes, nvm